docs/reference_guide.md

```diff
@@ -57,8 +57,8 @@ discussion to other files in /docs, the /tools/\*\_examples.txt files, or blog p
     - [14. `hardware`: Pre-defined Hardware Events](#14-hardware-pre-defined-hardware-events)
     - [15. `BEGIN`/`END`: Built-in events](#15-beginend-built-in-events)
     - [16. `watchpoint`/`asyncwatchpoint`: Memory watchpoints](#16-watchpointasyncwatchpoint-memory-watchpoints)
-    - [17. `kfunc`/`kretfunc`: Kernel Functions Tracing](#17-kfunckretfunc-kernel-functions-tracing)
-    - [18. `kfunc`/`kretfunc`: Kernel Functions Tracing Arguments](#18-kfunckretfunc-kernel-functions-tracing-arguments)
+    - [17. `kfunc`/`kretfunc` (alias: `fentry`/`fexit`): Kernel Functions Tracing](#17-kfunckretfunc-kernel-functions-tracing)
+    - [18. `kfunc`/`kretfunc` (alias: `fentry`/`fexit`): Kernel Functions Tracing Arguments](#18-kfunckretfunc-kernel-functions-tracing-arguments)
```

Comment:
Maybe leave the alias out of the section titles. Cuz we don't do that for the `u:` and `k:` stuff either. Also the link seems to be broken

Reply:
I wanted to make them easier to find but I guess people can always search/find.

Reply:
Yeah, I'm hoping to keep the headers terse. Grep will find it anyways